app/models/job.rb

```diff
@@ -38,7 +38,7 @@ def started_by?(user)
   end
 
   def can_be_stopped_by?(user)
-    started_by?(user) || user.is_admin?
+    started_by?(user) || user.is_admin? || user.is_admin_for?(project)
```

Comment:
Ah, I guess I misunderstood the difference between system admin and super admin. Cool.